.github/workflows/ci.yaml

```
@@ -127,4 +151,4 @@ jobs:
 
       - name: Run tests
         working-directory: ./ui
-        run: npm run test
+        run: npm run test
```

Whitespace error introduced (text files have to end with an empty line):
```suggestion
        run: npm run test

```